.github/workflows/correctness.yaml

```diff
@@ -23,6 +23,10 @@ jobs:
         run: poetry install
       - name: Pylint source
         run: poetry run pylint ./tds
+      - name: Pylint alembic
+        run: poetry run pylint ./migrate
+      - name: Pylint alembic versions
+        run: poetry run pylint ./migrate/versions
```

Comment:
Doesn't the top case also include the subdirectory? Do we need both?

Reply:
Pylint doesn't know about `migrate/versions` because it doesn't have an `__init__.py`. We could include inits but it would kind of be misleading about what the `migrate` directory is

Reply:
I can double check this behavior though

Reply:
i have double checked this behavior. Even `--recursive y` doesn't solve the issue. If we want to still change this, we can try `poetry run pylint $(git ls-files 'migrate/*.py')` (inspiration from [this post](poetry run pylint $(git ls-files 'migrate/*.py'))).

We could also add the `__init__.py` in but like I said before that would be misleading